js/views/endScreen.js

```diff
@@ -120,6 +134,10 @@ var EndScreen = createReactClass({
           <Icon {...this.props} icon="replay" style={actionIconStyle} />
         </button>
 
+        <div className={castPanelClass}>
+          <p>Connected to <span>{this.props.controller.state.cast.device}</span></p>
+        </div>
+
```

Comment:
There are 3 identical divs
```
<div className={castPanelClass}>
        <p>Connected to <span>{this.props.controller.state.cast.device}</span></p>
</div> 
```

in 3 different files (pause, play, end screen). Maybe it would be better to put these blocks in one file and to render this block?

Reply:
Yeah, I was thinking the same....I just follow the same that we had for other elements like the panel that shown the title and description on the start, pause and end screens...
Maybe I can move this to a separate component.